airbyte-integrations/connectors/destination-redshift/src/main/java/io/airbyte/integrations/destination/redshift/RedshiftDestination.java

```diff
@@ -25,45 +25,50 @@
 package io.airbyte.integrations.destination.redshift;
 
 import com.fasterxml.jackson.databind.JsonNode;
-import com.google.common.collect.ImmutableMap;
-import io.airbyte.commons.json.Jsons;
 import io.airbyte.integrations.base.Destination;
+import io.airbyte.integrations.base.DestinationConsumer;
 import io.airbyte.integrations.base.IntegrationRunner;
-import io.airbyte.integrations.destination.jdbc.AbstractJdbcDestination;
-import java.util.Optional;
+import io.airbyte.protocol.models.AirbyteConnectionStatus;
+import io.airbyte.protocol.models.AirbyteMessage;
+import io.airbyte.protocol.models.ConfiguredAirbyteCatalog;
+import io.airbyte.protocol.models.ConnectorSpecification;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class RedshiftDestination extends AbstractJdbcDestination implements Destination {
+public class RedshiftDestination implements Destination {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(RedshiftDestination.class);
 
-  public static final String DRIVER_CLASS = "com.amazon.redshift.jdbc.Driver";
-
-  public RedshiftDestination() {
-    super(DRIVER_CLASS, new RedshiftSQLNameTransformer(), new RedshiftSqlOperations());
+  public static void main(String[] args) throws Exception {
+    final Destination destination = new RedshiftDestination();
+    LOGGER.info("starting destination: {}", RedshiftDestination.class);
+    new IntegrationRunner(destination).run(args);
+    LOGGER.info("completed destination: {}", RedshiftDestination.class);
   }
 
   @Override
-  public JsonNode toJdbcConfig(JsonNode redshiftConfig) {
-    final String schema = Optional.ofNullable(redshiftConfig.get("schema")).map(JsonNode::asText).orElse("public");
+  public DestinationConsumer<AirbyteMessage> write(JsonNode config, ConfiguredAirbyteCatalog catalog) throws Exception {
```

Comment:
@cgardens just a skeleton - does this seem like a reasonable approach to support both destinations?

Reply:
yeah. this seems reasonable to me for keeping a clear separation.

Reply:
should we just dependency inject the 2 different destinations to make testing easier?

```
  public RedshiftDestination() {
     this(new RedshiftInsertDestination(), new RedshiftCopyDestination());
  }

  public RedshiftDestination(Destination insertDestination, Destination copyDestination) {
      ...etc
  } 
```